cpp/src/bitmask/null_mask.cu

```diff
@@ -222,6 +282,42 @@ cudf::size_type count_unset_bits(bitmask_type const *bitmask, size_type start,
   return (num_bits - detail::count_set_bits(bitmask, start, stop, stream));
 }
 
+// Create a bitmask from a vector of column views
+void concatenate_masks(std::vector<column_view> const &views,
+    bitmask_type * dest_mask,
+    cudaStream_t stream) {
+  using CDViewPtr =
+    std::unique_ptr<column_device_view,
+                    std::function<void(column_device_view*)>>;
+  std::vector<CDViewPtr> cols;
+  thrust::host_vector<column_device_view> deviceViews;
+
+  thrust::host_vector<size_type> view_offsets(1, 0);
+  for (auto &v : views) {
+    cols.emplace_back(column_device_view::create(v, stream));
+    deviceViews.push_back(*(cols.back()));
+    view_offsets.push_back(v.size());
+  }
+  thrust::inclusive_scan(thrust::host,
+      view_offsets.begin(), view_offsets.end(),
+      view_offsets.begin());
+
+  rmm::device_vector<column_device_view> dViews = deviceViews;
+  rmm::device_vector<size_type> dOffsets = view_offsets;
```

Comment:
```suggestion
  rmm::device_vector<size_type> d_offsets{view_offsets};
```

Reply:
[See](https://github.com/rapidsai/cudf/pull/3356#discussion_r348275226)

Reply:
This was more about the Hungarian Camel Case spelling than the assignment. Please at least spell it `d_offsets`.

Reply:
Fixed